tools/kagami/src/main.rs

```diff
@@ -635,95 +683,64 @@ mod docs {
 }
 
 mod tokens {
-    use std::collections::HashMap;
-
-    use clap::ArgEnum;
-    use color_eyre::{
-        eyre::{bail, eyre, WrapErr},
-        Result,
-    };
-    use iroha_permissions_validators::{
-        private_blockchain::register::CanRegisterDomains,
-        public_blockchain::PredefinedPermissionToken,
-    };
-    use iroha_schema::{IntoSchema, Metadata};
+    use color_eyre::{eyre::WrapErr, Result};
 
     use super::*;
 
     #[derive(StructOpt, Debug, Clone, Copy)]
-    pub struct Args {
-        #[structopt(arg_enum, default_value = "public")]
-        /// Whether to list private or public blockchain tokens
-        blockchain: Blockchain,
-    }
+    pub struct Args;
 
-    #[derive(ArgEnum, Debug, Clone, Copy)]
-    pub enum Blockchain {
-        Private,
-        Public,
+    pub fn permission_token_definitions() -> Result<Vec<PermissionTokenDefinition>> {
```

Comment:
you could return array. We have a ticket to fix this?

Reply:
Did it with array